austation/code/datums/components/crafting/recipes.dm

```diff
@@ -1,12 +1,21 @@
-/datum/crafting_recipe/makeshift_gun
-	name = "Makeshift pistol"
-	result = /obj/item/gun/ballistic/automatic/pistol/aumakeshift
-	reqs = list(/obj/item/weaponcrafting/receiver = 1,
-				/obj/item/pipe = 1,
-				/obj/item/weaponcrafting/stock = 1,
-				/obj/item/stack/packageWrap = 5,
-				/obj/item/stack/rods = 6)
-	tools = list(TOOL_SCREWDRIVER)
-	time = 100
-	category = CAT_WEAPONRY
-	subcategory = CAT_WEAPON
+/datum/crafting_recipe/femur_breaker
+	name = "Femur Breaker"
+	result = /obj/structure/femur_breaker
+	time = 150
+	reqs = list(/obj/item/stack/sheet/iron = 20,
+		        /obj/item/stack/cable_coil = 30)
```

Comment:
can you not make it require a little more than this?

Reply:
Now costs 25 iron, 10 plasteel and 60 cable.